nflog/nflog.go

```diff
@@ -154,76 +151,6 @@ func newMetrics(r prometheus.Registerer) *metrics {
 	return m
 }
 
-// Option configures a new Log implementation.
-type Option func(*Log) error
-
-// WithRetention sets the retention time for log st.
-func WithRetention(d time.Duration) Option {
-	return func(l *Log) error {
-		l.retention = d
-		return nil
-	}
-}
-
-// WithNow overwrites the function used to retrieve a timestamp
-// for the current point in time.
-// This is generally useful for injection during tests.
-func WithNow(f func() time.Time) Option {
-	return func(l *Log) error {
-		l.now = f
-		return nil
-	}
-}
```

Comment:
All of this plumbing and the `now       func() time.Time` on the manager struct was only to help with tests -- this is all replaced by using `clock.Clock`